tests/data/dataset_test.py

```diff
@@ -38,9 +38,9 @@ def test_as_arrays(self):
         dataset.index_instances(self.vocab)
         padding_lengths = dataset.get_padding_lengths()
         arrays = dataset.as_arrays(padding_lengths)
+        text1 = arrays["text1"]["words"]
```

Comment:
Why did you change the namespace from "tokens" to "words" in the test?  I think it'd be better to have it use the defaults - reading through this test might give you the wrong impression about how you should access the arrays.